libimu.c

```diff
@@ -71,6 +75,21 @@ int init_mpu(int sample_rate, int yaw_mix_factor)
 	return ret;
 }
 
+int enableFusion() {
```

Comment:
enableAccelerometerFusion() already calls printf so all these lib wrappers need to do is 

int enableFusion(void) {
return enableAccelerometerFusion();
}

also, the functions that take not parameters are all missing a void parameter. No parameter means it could take any number of parameters of unknown types in C and it makes the compiler generate a lot more code per function.